test/unit/tags/case_tag_unit_test.rb

```diff
@@ -82,6 +82,26 @@ def test_case_when_with_or
     end
   end
 
+  def test_case_when_empty
```

Comment:
I'm only adding a test for `empty` as `blank` requires activesupport in order to render as we expect, however both have parity with `lax`